src/components/NavigationMobile.tsx

```diff
@@ -48,40 +40,79 @@ type NavData = (
 )[]
 
 function NavList({ navData }: { navData?: NavData | null }) {
+  const theme = useTheme()
+
   if (!navData) {
     return null
   }
 
   return (
-    <>
-      {navData.map((navItem) => {
+    <div className="flex flex-col gap-medium">
+      {navData.map((navItem, navIndex) => {
         if (!navItem) {
           return null
         }
         if (isEmpty(navItem?.subnav)) {
           return (
-            <MobileMainLink
+            <Button
               key={navItem?.id}
               {...(navItem?.link?.url ? { href: navItem?.link.url } : {})}
+              as="a"
+              secondary
+              endIcon={<ArrowRightIcon />}
+              style={{
+                justifyContent: 'space-between',
+                padding: theme.spacing.medium,
+                borderColor: theme.colors.border,
+              }}
             >
               {navItem?.link?.title}
-            </MobileMainLink>
+            </Button>
           )
         }
 
         return (
-          <section
+          <Accordion
+            label={navItem?.link?.title}
             key={navItem.id}
-            className="mb-medium"
+            // @ts-ignore
+            style={{ backgroundColor: theme.colors['fill-two'] }}
           >
-            {navItem?.link?.title ? (
-              <MenuHeading>{navItem?.link?.title}</MenuHeading>
-            ) : null}
-            <NavList navData={navItem?.subnav} />
-          </section>
+            {navItem.subnav?.map((subnavItem) => {
+              if (!subnavItem) {
+                return null
+              }
+
+              if (!navIndex) {
```

Comment:
nit but this should probably just check if the navItem title is "product" instead of index being 0